Library/Homebrew/install.rb

```diff
@@ -32,12 +32,16 @@ def perform_build_from_source_checks(all_fatal: false)
     def check_prefix
       if (Hardware::CPU.intel? || Hardware::CPU.in_rosetta2?) &&
          HOMEBREW_PREFIX.to_s == HOMEBREW_MACOS_ARM_DEFAULT_PREFIX
-        configuration = if Hardware::CPU.in_rosetta2?
-          "under Rosetta 2"
+        if Hardware::CPU.in_rosetta2?
+          configuration = "under Rosetta 2"
```

Comment:
This variable feels like it could be inlined in both places if it's not being used outside of the string.

Reply:
done